qcli/bsp2svg/converter.py

```diff
@@ -31,52 +31,67 @@ def convert(bsp_file, svg_file, args):
     """
     print(f'Reading {os.path.basename(bsp_file)}')
     bsp_file = Bsp.open(bsp_file)
+    projection_axis = args.projection_axis
+    
+    # Filter faces
+    faces = [face for model in bsp_file.models for face in model.faces]
+    ignore_textures = ['clip', 'hint', 'trigger'] + args.ignore
+    faces = list(filter(lambda f: not (f.texture_name.startswith('sky') or f.texture_name in ignore_textures), faces))
 
     # Determine map bounds
-    vs = [vertex[:] for model in bsp_file.models for face in model.faces for vertex in face.vertexes]
+    vs = [vertex[:] for face in faces for vertex in face.vertexes]
```

Comment:
Why did you break apart the nested list comprehension? Is it too tricky to understand/reckon about?

Reply:
ah good point. this part will help with the next PR about slicing, where the `faces` variable will be filtered again based on their `plane`. it's not necessary, but it will help with the future PR

Reply:
Gotcha. I understand better now. I'm totally fine with the `faces` list. 👍